src/App.js

```diff
@@ -68,7 +69,7 @@ function App() {
         <Route exact path="/projects/:id/:title">
           <ProjectDetails />
         </Route>
-        <Route path="/:userName" exact>
+        <Route path="/:userId/:userName" exact>
```

Comment:
You may want to change these routes to start with `/user`, e.g. `/user/:userId/:userName` - that way you can avoid conflicts with other routes.

Reply:
Makes sense, thank you!